cmd/main.go

```diff
@@ -130,8 +133,10 @@ func Main(version string) {
 }
 
 func init() {
-	if os.Getenv("CN_REGISTRY") == "redhat" {
-		imageName = "registry.access.redhat.com/rhceph/rhceph-3-rhel7"
+	if configurationFile = readConfigFile(); len(configurationFile) > 0 {
+		fmt.Printf("Using %s as configuration file\n", configurationFile)
```

Comment:
Use `log.Println` instead so we are consistent with the existing code.

Reply:
It's a bit 'annoying' to see this message each time we run the CLI.
Maybe we could have an option to suppress it in the config file?

Reply:
Printf have the advantage to ease the formating. It that critcial to have only println ?

Reply:
Regarding my experience during the devel, as the behavior changes when you have a config file or not, and as the configuration file could be in several places, that is very difficult to anticipate if the bad behavior is due to the presence/lack/wrong config file.
That's why I put this at every start, so it's explicit. I know its verbose but useful.

Reply:
@leseb This is the last item to handle.